R/utils.R

```diff
@@ -1,3 +1,17 @@
+#' Citation for Spatial Equity Data Tool API Data
+#'
+#' @name setd-citation
```

Comment:
We want this to be sedt-citation (t after d)

Reply:
Fixed. Not my first time with this specific typo, regrettably!